lib/active_merchant/billing/gateways/paysafe.rb

```diff
@@ -0,0 +1,282 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class PaysafeGateway < Gateway
+      self.test_url = 'https://api.test.paysafe.com'
+      self.live_url = 'https://api.paysafe.com'
+
+      self.supported_countries = %w(FR)
+      self.default_currency = 'EUR'
+      self.supported_cardtypes = %i[visa master american_express discover]
+
+      self.homepage_url = 'https://www.paysafe.com/'
+      self.display_name = 'Paysafe'
+
+      def initialize(options = {})
+        requires!(options, :username, :password, :account_id)
+        super
+      end
+
+      # Purchase requests for this gateway happen at the same endpoint as `authorize`
+      # but they include the `settleWithAuth` parameter
```

Comment:
I don't think you need this comment, or the ones on line 51 and 59.

Reply:
Removed!